hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/cache/CacheManager.java

```diff
@@ -56,7 +56,7 @@ public static boolean cacheEnableMetrics(String name, boolean enabled) {
         return cache.enableMetrics(enabled);
     }
 
-    public CacheManager() {
+    private CacheManager() {
```

Comment:
emm? why this commit appear again? a little confused

Reply:
thx, it has been corrected.